.github/workflows/cypress_tests.yml

```diff
@@ -0,0 +1,248 @@
+name: "[Workflow] Cypress Tests"
+
+defaults:
+  run:
+    shell: bash
+
+on:
+  workflow_call:
+    inputs:
+      account_id:
+        description: "AWS account ID containing the role to assume for boto commands"
+        required: true
+        type: string
+      front_sg_name:
+        description: "Name of the security group used by the front load balancer"
+        required: true
+        type: string
+      admin_sg_name:
+        description: "Name of the security group used by the admin load balancer"
+        required: true
+        type: string
+      admin_url:
+        description: "URL of the admin frontend"
+        required: true
+        type: string
+      front_url:
+        description: "URL of the frontend"
+        required: true
+        type: string
+
+    secrets:
+      AWS_ACCESS_KEY_ID_ACTIONS:
+        required: true
+      AWS_SECRET_ACCESS_KEY_ACTIONS:
+        required: true
+
+jobs:
+  cypress_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # pin@v3
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838 # pin@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_ACTIONS }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_ACTIONS }}
+          aws-region: eu-west-1
+          role-to-assume: arn:aws:iam::${{ inputs.account_id }}:role/opg-lpa-ci
+          role-duration-seconds: 900
```

Comment:
This will need to be longer than 15 minutes as the test suite takes longer than that to run. The ingress management at the end of the run will fail because the session ended.